components/Footer.tsx

```diff
@@ -165,40 +181,72 @@ const Footer = () => {
           </FooterList>
         </FooterCategory>
         <FooterCategory>
-          <FooterHeader>Technology</FooterHeader>
+          <FooterHeader>
+            <DeveloperBoardIcon style={{ marginLeft: -5, marginRight: 10 }} />
+            Technology
+          </FooterHeader>
           <FooterList>
             <FooterItem href="https://docs.dahliaos.io/os/linux">
               How it works
             </FooterItem>
-            <FooterItem href="#">The goal</FooterItem>
-            <FooterItem href="#">Design</FooterItem>
-            <FooterItem href="#">UI Modularity</FooterItem>
+            <FooterItem href="https://docs.dahliaos.io/pangolin/pangolin">
+              Pangolin
+            </FooterItem>
+            <FooterItem href="https://docs.dahliaos.io/os/fimage">
+              FImage
+            </FooterItem>
+            <FooterItem href="https://docs.dahliaos.io/developer/packaging">
+              Packaging
+            </FooterItem>
+            <FooterItem href="https://docs.dahliaos.io/developer/roadmap">
+              The goal
+            </FooterItem>
           </FooterList>
         </FooterCategory>
         <FooterCategory>
-          <FooterHeader>Documentation</FooterHeader>
+          <FooterHeader>
+            <ArticleIcon style={{ marginLeft: -5, marginRight: 10 }} />
+            Documentation
+          </FooterHeader>
           <FooterList>
-            <FooterItem href="https://docs.dahliaos.io/os/linux">
+            <FooterItem href="https://docs.dahliaos.io/build/buildroot">
               Compiling
             </FooterItem>
             <FooterItem href="https://docs.dahliaos.io/install/efi">
               Installing
             </FooterItem>
-            <FooterItem href="https://docs.dahliaos.io/install/efi">
-              Running
-            </FooterItem>
             <FooterItem href="https://docs.dahliaos.io/hardware/support">
               Supported devices
             </FooterItem>
-            <FooterItem href="#">Linux vs. Zircon</FooterItem>
+            <FooterItem href="https://docs.dahliaos.io/os/faq">FAQ</FooterItem>
```

Comment:
For all these footer items, can we have it

On dark mode:
dark color on initial state
Lighter color on hover state

and on light mode
lighter color on initial state
darker color on hover state